assets/css/style.css

```diff
@@ -19,22 +17,18 @@ h1, h2, h3, h4, h5, h6 {
 
 .container {
     width: 100%;
-    max-width: 1200px;
     margin: 0 auto;
 }
 
-.intro {
-    width: 100%;
-    height: 100vh;
-}
 
 
 .header {
     width: 100%;
-    height: 70px;
-    padding-top: 15px;
+    height: 9%;

```

Comment:
nah, u header should be stretched depending on what content is inside. u can set a min-height, and not in %, in rem